src/test/protocol/STTx_test.cpp

```diff
@@ -1344,22 +1344,22 @@ class STTx_test : public beast::unit_test::suite
             0x73, 0x00, 0x81, 0x14, 0x00, 0x10, 0x00, 0x00, 0x00, 0x00, 0x00,
             0x00, 0x65, 0x24, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
             0xe5, 0xfe, 0xf3, 0xe7, 0xe5, 0x65, 0x24, 0x00, 0x00, 0x00, 0x00,
-            0x20, 0x1e, 0x00, 0x6f, 0x00, 0x00, 0x20, 0x1f, 0x03, 0xf6, 0x00,
```

Comment:
what's  changing this for?

Reply:
I took this change from Howard's MPT branch. One of the tests fails without it.